runtime/fundamentals/security.md

```diff
@@ -95,6 +95,11 @@ By default, Deno will not generate a stack trace for permission requests as it
 comes with a hit to performance. Users can enable stack traces with the
 `DENO_TRACE_PERMISSIONS` environment variable to `1`.
 
+### Configuration File
+
```

Comment:
I think we should call out this is available in 2.5+. CC @philhawksworth @thisisjofrank